api/v1alpha1/fenceagentsremediation_webhook_test.go

```diff
@@ -11,58 +11,118 @@ var _ = Describe("FenceAgentsRemediation Validation", func() {
 
 	Context("creating FenceAgentsRemediation", func() {
 
-		When("agent name match format and binary", func() {
-			It("should be accepted", func() {
-				far := getTestFAR(validAgentName)
-				_, err := far.ValidateCreate()
-				Expect(err).ToNot(HaveOccurred())
+		Context("with ResourceDeletion strategy", func() {
+
+			When("agent name match format and binary", func() {
+				It("should be accepted", func() {
+					far := getTestFAR(validAgentName)
+					Expect(far.ValidateCreate()).Error().NotTo(HaveOccurred())
```

Comment:
nit: Expect(far.ValidateCreate()).To(Succeeed()) is easier

Reply:
ValidateCreate() returns "(admission.Warnings, error)". So this way doesn't work well.

Reply:
you're right, sorry, I forgot the new signature of the validation methods